docker-entrypoint.sh

```diff
@@ -50,14 +78,64 @@ if [ "$1" = 'rabbitmq-server' ]; then
 		done
 		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			      {loopback_users, []}
+		EOF
+
+		if [ "$RABBITMQ_MANAGEMENT_ENABLED" ]; then
+			cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
+				    ]
+				  },
+				  { rabbitmq_management, [
+				      { listener, [
+			EOF
+
+			if [ "${use_ssl}" = "yes" ]; then
+				cat >> /etc/rabbitmq/rabbitmq.config <<-EOS
+				      { port, 15671 }, 
+				      { ssl, true },
+				      { ssl_opts, [ 
+				          { certfile,   "$RABBITMQ_SSL_CERT_FILE" },
+				          { keyfile,    "$RABBITMQ_SSL_KEY_FILE" },
+				          { cacertfile, "$RABBITMQ_SSL_CA_FILE" },
+				      { verify,   verify_none },
+				      { fail_if_no_peer_cert, false } ] } ] }
+				EOS
+			else
+				cat >> /etc/rabbitmq/rabbitmq.config <<-EOS
+				        { port, 15672 },
+				        { ssl, false } 
+				        ]
+				      }
+				EOS
+			fi
+		fi
+
+		cat >> /etc/rabbitmq/rabbitmq.config <<-'EOF'
 			    ]
 			  }
 			].
 		EOF
 	fi
 
+	if [ "${use_ssl}" = "yes" ]; then
+		# Create combined cert
+		cat ${RABBITMQ_SSL_CERT_FILE} ${RABBITMQ_SSL_KEY_FILE} > /tmp/combined.pem
+		chmod 0400 /tmp/combined.pem
+
+		# More ENV vars for make clustering happiness
+		# we don't handle clustering in this script, but these args should ensure
+		# clustered SSL-enabled members will talk nicely
+		export ERL_SSL_PATH=`erl -eval 'io:format("~p", [code:lib_dir(ssl, ebin)]),halt().' -noshell`
```

Comment:
Would much prefer to see the `$(` form here rather than backticks -- it's more obvious that it's shelling out, and it has the benefit of being able to nest sanely if needed for future expansion, ie:
```bash
export ERL_SSL_PATH="$(erl -eval 'io:format("~p", [code:lib_dir(ssl, ebin)]),halt().' -noshell)"
```